src/main/java/com/jcabi/dynamodb/maven/plugin/CreateTablesMojo.java

```diff
@@ -112,231 +80,14 @@
-        } catch (final ClassCastException ex) {
-            result = Long.parseLong(json.getString(name));
-        }
-        return result;
-    }
-
-    /**
-     * Get projection JSON element.
-     * @param json JSON input
-     * @return Projection
-     */
-    private Projection projection(final JsonObject json) {
-        final JsonObject projn = json.getJsonObject("Projection");
-        final Projection projection = new Projection()
-            .withProjectionType(projn.getString("ProjectionType"));
-        final Collection<String> nonkeyattrs = new LinkedList<String>();
-        if (projn.containsKey("NonKeyAttributes")) {
-            for (final JsonValue nonkey
-                : projn.getJsonArray("NonKeyAttributes")) {
-                nonkeyattrs.add(nonkey.toString());
-            }
-            projection.setNonKeyAttributes(nonkeyattrs);
-        }
-        return projection;
-    }
-
-    /**
-     * Get key schema elements.
-     * @param json JSON input
-     * @return Key schema elements
-     */
-    private Collection<KeySchemaElement> keySchema(final JsonObject json) {
-        final Collection<KeySchemaElement> keys =
-            new LinkedList<KeySchemaElement>();
-        final JsonArray schema = json.getJsonArray("KeySchema");
-        for (final JsonValue value : schema) {
-            final JsonObject element = (JsonObject) value;
-            keys.add(
-                new KeySchemaElement(
-                    element.getString("AttributeName"),
-                    element.getString("KeyType")
-                )
-            );
-        }
-        return keys;
-    }
-
-    /**
-     * Reads a file's contents into a JsonObject.
-     * @param file The path of the file to read
-     * @return The JSON object
-     * @throws MojoFailureException If there is an execution failure.
-     */
-    private JsonObject readJson(final String file)
-        throws MojoFailureException {
-        InputStream stream = null;
-        JsonObject json = null;
-        try {
-            stream = new FileInputStream(file);
-            json = Json.createReader(stream).readObject();
+            new Tables(
+                this.tables, this.endpoint, this.tcpPort(), this.key,
+                this.secret
+            )
+                .create(instances);
         } catch (final IOException ex) {
-            throw new MojoFailureException(
-                "Failed to read table definition", ex
-            );
-        } finally {
-            if (stream != null) {
-                try {
-                    stream.close();
-                } catch (final IOException ex) {
-                    Logger.error(
-                        this,
-                        "Failed to close stream with message %s",
-                        ex.getMessage()
-                    );
-                }
-            }
+            throw new MojoFailureException(ex.getMessage(), ex);
```

Comment:
@piddubnyi please read http://www.yegor256.com/2015/12/01/rethrow-exceptions.html and add proper context